### ISSUE TITLE
German UI translation missing / not showing up

### ISSUE THREAD

**Issue by ClemensGruber** [polimediaupv/paella-ethz#27]
The Paella UI language is auto-detected from the browser language afaik? So my browser in the screenshot is German / DE but the Paella UI still EN, do we have 

- no translation (DE, ES) for the current player version or 
- is it not configured / activated or 
- is the auto-detected not working in my case?

<img width="720" alt="2023-03-10 17_36_47-Einstellungen" src="https://user-images.githubusercontent.com/36139413/224372690-9f8ffbef-ac3e-4e5c-a3f7-8352d39b2f9f.png">

_Originally posted by @ClemensGruber in https://github.com/polimediaupv/paella-ethz/issues/7#issuecomment-1464074578_

**Comment by miesgre** [comment]
There are no translations for German in paella.
We don't know german language (sorry). But If someone make a pull request for german (or any other language) we will apreciate it.

- https://github.com/polimediaupv/paella-core/tree/main/src/i18n
- https://github.com/polimediaupv/paella-basic-plugins/tree/main/src/i18n
- https://github.com/polimediaupv/paella-slide-plugins/tree/main/src/i18n
- https://github.com/polimediaupv/paella-zoom-plugin/tree/main/src/i18n

**Comment by ClemensGruber** [comment]
I wonder how this could happen :) or how it is managed in current (German) Opencast deployments ...

**Comment by miesgre** [comment]
Yes, paella 6 has localization files for some languages: https://github.com/polimediaupv/paella/tree/develop/localization
Paella 7 is a completely new player and currently only have english and spanish

Paella 7 is splited in different packages, and there are some localizations files (one for package):
- https://github.com/polimediaupv/paella-core/tree/main/src/i18n
- https://github.com/polimediaupv/paella-basic-plugins/tree/main/src/i18n
- https://github.com/polimediaupv/paella-slide-plugins/tree/main/src/i18n
- https://github.com/polimediaupv/paella-zoom-plugin/tree/main/src/i18n

**Comment by miesgre** [comment]
> Hmm, seems that we can not recycle anything from https://github.com/polimediaupv/paella/blob/develop/localization/dictionary_de.json there are no lables / tooltips for buttons in this file, this is the only (old) translatin file we have or is some other text hardcoded on an oterh place we can re-use?

Translations from paella 6 can be obtained from https://github.com/opencast/opencast/tree/develop/modules/engage-paella-player/src/main/paella-opencast/plugins. There are some folders (from plugins). Inside these plugins folders there is a `locatization/de-DE.json` file . Example: https://github.com/opencast/opencast/blob/develop/modules/engage-paella-player/src/main/paella-opencast/plugins/es.upv.paella.opencast.loader/localization/de-DE.json

But I'm not sure we can reuse the old paella 6 labels.

**Comment by oas777** [comment]
PS: Files to translate:
https://github.com/polimediaupv/paella-core/tree/main/src/i18n
https://github.com/polimediaupv/paella-basic-plugins/tree/main/src/i18n
https://github.com/polimediaupv/paella-slide-plugins/tree/main/src/i18n
https://github.com/polimediaupv/paella-zoom-plugin/tree/main/src/i18n

**Comment by oas777** [comment]
No priority at this time.
